package-lock.json

```diff
@@ -1,8 +1,474 @@
 {
   "name": "hsd",
   "version": "2.4.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
Did you intend to commit package-lock? Usually we don't, right? I've seen this new lockfile version, need to learn about it -- can it be a separate PR?

Reply:
We have package-lock committed and we usually commit (that's some minimal checksum for you), but yeah I used latest node/npm which apparently upgrades package-lock version with new format. So it's better to do in separate PR.